VndbSharp/VndbClient.cs

```diff
@@ -145,7 +145,12 @@ public VndbClient(String username, SecureString password)
 			=> await this.SendRequestInternalAsync<RootObject<Wishlist>>(Constants.GetWishlistCommand, flags, filters, options)
 				.ConfigureAwait(false);
 
-		public async Task<String> DoRawAsync(String command)
+        public async Task<Boolean> SetVotelistAsync(UInt32 id, Object data)
```

Comment:
Please use the correct type for Data, and a proper variable name for it

In this case it would be a `Byte? vote`, since the value will never be below 0, but never above 100 (while Byte can go to 255)